showcase/src/Layouts/Stories.elm

```diff
@@ -0,0 +1,122 @@
+module Layouts.Stories exposing (stories, update)
+
+import Element exposing (Element, fill, maximum)
+import Layouts.Model as LayoutsModel
+import Layouts.Msg as LayoutsMsg
+import Model exposing (Model)
+import Msg exposing (Msg)
+import PluginOptions exposing (defaultWithoutMenu)
+import Return exposing (Return)
+import Tables.Book exposing (Book, books)
+import UI.Internal.NavigationContainer
+import UI.Layout.SplitSelectable as SplitSelectable
+import UI.ListView as ListView exposing (ListView)
+import UI.Palette as Palette exposing (brightnessLighter, tonePrimary)
+import UI.RenderConfig exposing (RenderConfig)
+import UI.SummaryListItem as Summary
+import UI.Text as Text
+import UIExplorer exposing (storiesOf)
+import Utils exposing (ExplorerStory, ExplorerUI, prettifyElmCode, storyWithModel)
+
+
+update : LayoutsMsg.Msg -> LayoutsModel.Model -> Return LayoutsMsg.Msg LayoutsModel.Model
+update msg model =
+    case msg of
+        LayoutsMsg.NoOp ->
+            ( model, Cmd.none )
+
+        LayoutsMsg.Select book ->
+            ( { model | selected = Just book }, Cmd.none )
+
+
+stories : RenderConfig -> ExplorerUI
+stories cfg =
+    storiesOf
+        "Layouts"
+        [ demo cfg
+        ]
+
+
+demo : RenderConfig -> ExplorerStory
+demo renderConfig =
+    storyWithModel
```

Comment:
Does it make sense to add a border color or something like that just to indicate what is the exact component? Maybe it could help people that are not familiar with this UI

Reply:
to `storyWithModel` or only this story?

Reply:
Well, that's a quite good question, I this problem affects more layouts, "big area" components", so maybe we could create another storyWithModel function with the border or adding a flag to this function enabling/disabling the border